app/adapters/application.js

```diff
@@ -1,6 +1,5 @@
 import DS from "ember-data";
 
-// let underscore = Ember.String.underscore;
 var host='http://localhost:3000';
```

Comment:
Optional: seems like host was never changed after it was declared. using const will be better for readability.